test/run.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { runCLI } = require('jest')
+const jestConfig = require('../jest.config.js')
+
+const sessionPlugins = ['fastify-secure-session', '@fastify/session']
+
+async function runTests(sessionPlugin) {
+  process.env.SESSION_PLUGIN = sessionPlugin
+  const result = await runCLI(jestConfig, ['./'])
+  if (result.results.success) {
+    console.log(`Tests completed`)
+  } else {
+    console.error(`Tests failed`)
```

Comment:
I think we should stop the execution of the tests maybe?

Reply:
Not sure. Maybe we can let it to run so you can see failed tests from both runs?